circle.yml

```diff
@@ -1,4 +1,8 @@
 ---
+dependencies:
+  pre:
+    - sudo pip install docker-compose
```

Comment:
The version on CircleCI is a bit outdated, this ensures we get the latest version,